src/Layout/components/header.js

```diff
@@ -13,15 +12,21 @@ export default class ThankYou extends Component {
             <Navbar staticTop>
                 <Navbar.Header>
                     <Navbar.Brand>
-                        <Link to='/'>Home</Link>
+                        <Link to="/">Home</Link>
                     </Navbar.Brand>
                     <Navbar.Toggle />
                 </Navbar.Header>
                 <Navbar.Collapse>
                     <Nav>
-                        <NavItem eventKey={1} href="/donation-form">Donation form link</NavItem>
-                        <NavItem eventKey={2} href="/donation-form/thank-you">Thank You page link</NavItem>
-                        <NavItem eventKey={3} href='/search'>Search page link</NavItem>
```

Comment:
info: just curious why we seem to hope between <Link> and <NavItem> in this header. :)

Reply:
So NavItem was causing a full page reload (a regular anchor tag), Link however uses React router and the history API avoiding the full page reload

Reply:
Ah, good to know! Thanks.